sds_data_manager/stacks/ecr_stack.py

```diff
@@ -52,3 +52,13 @@ def __init__(
             ecr_authenticators.add_user(user)
 
         self.container_repo.apply_removal_policy(RemovalPolicy.DESTROY)
+
+        # Create ECR Policy
```

Comment:
If these other policies are needed can you add them over in the IalirtEC2 stack location since this isn't used directly here?
I'm not sure this is needed though, or maybe you could find a managed policy that enables the PULL for a specific repository easier?